src/Collection.php

```diff
@@ -367,7 +367,7 @@ public function frequency(): CollectionInterface
      * @template NewTKey
      * @template NewT
      *
-     * @param callable(...mixed): Iterator<NewTKey, NewT> $callable
+     * @param callable(mixed ...$parameters): iterable<NewTKey, NewT> $callable
```

Comment:
- the `...mixed` syntax is not supported by PHPStan, but the `mixed ...$parameters` is supported by both Psalm and PHPStan

- also note the collection can be instantiated with a callable returning any `iterable`, not just Iterator, as exemplified in the [SA check](https://github.com/loophp/collection/blob/master/tests/static-analysis/fromCallable.php#L144-L146)

Reply:
Great!